go.mod

```diff
@@ -0,0 +1,95 @@
+	github.com/jinzhu/inflection v0.0.0-20170102125226-1c35d901db3d // indirect
+	github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
+	github.com/jrallison/go-workers v0.0.0-20180112190529-dbf81d0b75bb
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
+	github.com/kr/pretty v0.2.1 // indirect
+	github.com/kr/text v0.2.0 // indirect
+	github.com/labstack/echo v2.2.0+incompatible
+	github.com/labstack/gommon v0.2.2-0.20170925052817-57409ada9da0 // indirect
+	github.com/lib/pq v1.0.0 // indirect
+	github.com/magiconair/properties v1.8.1 // indirect
+	github.com/mailru/easyjson v0.0.0-20180320131758-517203d186eb // indirect
+	github.com/mattn/go-colorable v0.1.7 // indirect
+	github.com/mattn/go-sqlite3 v1.14.4 // indirect
+	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b // indirect
+	github.com/mgutz/logxi v0.0.0-20161027140823-aebf8a7d67ab // indirect
+	github.com/mgutz/str v1.2.0 // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/nxadm/tail v1.4.5 // indirect
+	github.com/onsi/ginkgo v1.14.2
+	github.com/onsi/gomega v1.10.1
+	github.com/opentracing/opentracing-go v1.2.0
+	github.com/orfjackal/nanospec.go v0.0.0-20120727230329-de4694c1d701 // indirect
+	github.com/pierrec/lz4 v1.0.2-0.20171218195038-2fcda4cb7018 // indirect
+	github.com/pierrec/xxHash v0.1.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/poy/onpar v1.0.1 // indirect
+	github.com/rcrowley/go-metrics v0.0.0-20180125231941-8732c616f529 // indirect
+	github.com/satori/go.uuid v1.2.0
+	github.com/sirupsen/logrus v1.2.0
+	github.com/smartystreets/goconvey v1.6.4 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/spf13/viper v1.4.0
+	github.com/stretchr/testify v1.6.1 // indirect
+	github.com/uber/jaeger-client-go v2.25.0+incompatible
+	github.com/uber/jaeger-lib v2.4.0+incompatible // indirect
+	github.com/valyala/bytebufferpool v1.0.0 // indirect
+	github.com/valyala/fasttemplate v0.0.0-20170224212429-dcecefd839c4 // indirect
+	github.com/ziutek/mymysql v1.5.4 // indirect
+	golang.org/x/net v0.0.0-20200625001655-4c5254603344 // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/sys v0.0.0-20201101102859-da207088b7d1 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	gopkg.in/alexcesaro/statsd.v2 v2.0.0 // indirect
+	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
+	gopkg.in/inf.v0 v0.9.0 // indirect
+	gopkg.in/ini.v1 v1.62.0 // indirect
+	gopkg.in/mgo.v2 v2.0.0-20160818020120-3f83fa500528
+	gopkg.in/mgutz/dat.v2 v2.0.0-20171004160617-d76e4f81c4ef
+	gopkg.in/olivere/elastic.v5 v5.0.66
+	gopkg.in/stretchr/testify.v1 v1.2.2 // indirect
+	mellium.im/sasl v0.2.1 // indirect
+)
+
+replace github.com/codahale/hdrhistogram => github.com/HdrHistogram/hdrhistogram-go v0.0.0-20200919145931-8dac23c8dac1
```

Comment:
Is this replace for our usage? If yes, why not change it?

Reply:
This replace is for the jaeger dependecy